pkg/products/bamboo/helm_release.tf

```diff
@@ -59,4 +59,34 @@ data "kubernetes_service" "bamboo" {
     name      = "bamboo"
     namespace = kubernetes_namespace.bamboo.metadata[0].name
   }
+}
+
+resource "helm_release" "bamboo_agent" {
+  name       = "bamboo-agent"
+  namespace  = kubernetes_namespace.bamboo.metadata[0].name
+  repository = "https://atlassian.github.io/data-center-helm-charts"
```

Comment:
Would you please create a local variable for atlassian helm chart repository and use the variable here and in `helm_release.bamboo`